pkg/rbac/clusterScopedResources_test.go

```diff
@@ -0,0 +1,112 @@
+package rbac
+
+import (
+	"testing"
+	"time"
+
+	"github.com/driftprogramming/pgxpoolmock"
+	"github.com/golang/mock/gomock"
+)
+
+// Initialize cache object to use tests.
+func newResourcesListCache(t *testing.T) (*pgxpoolmock.MockPgxPool, Cache) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	mockPool := pgxpoolmock.NewMockPgxPool(ctrl)
+	return mockPool, Cache{
+		shared: sharedList{},
+		pool:   mockPool,
+	}
+}
+
+func Test_getResources_emptyCache(t *testing.T) {
+
+	mockpool, cache := newResourcesListCache(t)
```

Comment:
I'd name this `mock_cache` for clarity and match the tokenReview tests.

Reply:
Done.